libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/navigationview/NavigationViewContext.kt

```diff
@@ -50,12 +51,14 @@ internal class NavigationViewContext(
     val mapViewOwner = MapViewOwner()
     val mapStyleLoader = MapStyleLoader(context, options)
     val mapClickBehavior = MapClickBehavior()
+    val speedInfoBehavior = SpeedInfoBehavior()
```

Comment:
You can combine all behaviors into a single class if you want to avoid adding new arguments for `NavigationViewListenerRegistry`:
```kotlin
class NavigationViewBehaviors {
    val maneuverBehavior = ManeuverBehavior()
    val infoPanelBehavior = InfoPanelBehavior()
    val mapClickBehavior = MapClickBehavior()
    val speedInfoBehavior = SpeedInfoBehavior()
}
```